app/src/main/res/values-nl/strings.xml

```diff
@@ -1,66 +1,66 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">Eenvoudige Galerij</string>
+    <string name="app_name">Eenvoudige galerij</string>
     <string name="app_launcher_name">Galerij</string>
     <string name="edit">Bewerken</string>
     <string name="open_camera">Camera</string>
     <string name="hidden">(verborgen)</string>
     <string name="pin_folder">Map vastzetten</string>
     <string name="unpin_folder">Map losmaken</string>
-    <string name="pin_to_the_top">Pin to the top</string>
-    <string name="show_all">Platte weergave</string>
+    <string name="pin_to_the_top">Bovenaan vastzetten</string>
+    <string name="show_all">Alles weergeven</string>
     <string name="all_folders">Alle mappen</string>
     <string name="folder_view">Mapweergave</string>
     <string name="other_folder">Andere map</string>
-    <string name="show_on_map">Op kaart tonen</string>
-    <string name="unknown_location">Locatie onbekend</string>
-    <string name="no_map_application">Geen app voor kaarten gevonden</string>
-    <string name="no_camera_app_found">Geen app voor camera gevonden</string>
+    <string name="show_on_map">Weergeven op kaart</string>
```

Comment:
No (verb at end for consistency)

Reply:
That's a matter of opinion, but 'weergeven' is more common here in the Netherlands.

Reply:
True, but it's also a matter of concistency now. Furthermore, I think "kaart" is the key word here, that indicates where this will take you, not "weergeven". So putting that in front will improve interpretation.

Reply:
Usually one decides on an action to take. That's also why Apple translates everything with actions at the beginning, for example "Bewerk instellingen" (and no, I'm not an Apple fan, I'm just giving an example). But most other software also uses interpretations where actions are in front.